app/src/pages/api/attachment/upload-url.ts

```diff
@@ -1,28 +1,20 @@
-import S3 from 'aws-sdk/clients/s3';
 import { NextApiRequest, NextApiResponse } from 'next';
+import { S3DbClient } from '../../../utils/clients/S3DbClient';
+import { IS3Object } from '../../../models/data/IS3Object';
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse) {
-    console.log('Using BUCKET_NAME:', process.env.BUCKET_NAME);
-    const s3 = new S3({
-        apiVersion: '2006-03-01',
-        accessKeyId: process.env.CL_AWS_ACCESS_KEY,
-        secretAccessKey: process.env.CL_AWS_SECRET_ACCESS_KEY,
-        region: process.env.AWS_REGION,
-    });
+    const s3DbClient = new S3DbClient();
 
-    const MEGABYTE = 10 ** 6;
+    const s3Object: Omit<IS3Object, 'url'> = {
+        objectId: req.query.file as string,
+        fileType: req.query.fileType as string,
```

Comment:
Should we include validation on this to make sure that `req.query.file` and `req.query.fileType` exist and are strings? So that malicious or invalid input won't break our app :) I think the guest book apis could have good examples of hthis